### PR TITLE
XNet machine textures & Better Foliage short grass & two smail mod

### PR DESCRIPTION
**Pull request by mgdgfbhevdyeh**
XNet many Templates form RFTOOLS texture

### CODE REVIEW

File: Better_Foliage/mod.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "mod_id": "betterfoliage",
+  "mod_name": "Better Foliage",
+  "mod_dir": "/BetterFoliage/",
```
Comment (alfw) [inline Better_Foliage/mod.json]: The ``mod_dir`` need to match the dir where the texutres are stored. In this it has to be ``"mod_dir": "/Better_Foliage/",``

File: Flour_Power/mod.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "mod_id": "Flour_Power",
-  "mod_name": "FlourPower",
+  "mod_name": "Flour Power",
   "mod_dir": "/FlourPower/",
```
Comment (alfw) [inline Flour_Power/mod.json]: The ``mod_dir`` need to match the dir where the texutres are stored. In this it has to be "mod_dir": "/Flour_Power/",

**Review by alfw — CHANGES_REQUESTED** [review]
Hello,
Those changes is needed to make our scripts happy. This should not be merged until fixed.

**Comment by mgdgfbhevdyeh** [comment]
OK I know :-D
sorry my is bad for write mod.json

**Review by alfw — APPROVED** [review]

**Pull request merged**